### ISSUE TITLE
Configurations are not quite user agnostic

### ISSUE THREAD

**Issue by ldb** [saimoomedits/dotfiles#4]
First of all, this rice looks awesome from the screenshots!
I wanted to try it out but am encountering some problems, for example when trying to launch rofi. The username `saimoom` is hardcoded in some paths, resulting some config files not being found.
In `.config/awesome` I found:

```
awesome/config/bling.lua
42:        image = gears.color.recolor_image("/home/saimoom/wallpapers/plain.png", beautiful.bg_color),

awesome/layout/widgets/dashboard/profile.lua
37:    markup = "<span foreground='" .. beautiful.fg_color .. "BF" .. "'>Saimoom</span>",

awesome/layout/widgets/dashboard/ss.lua
57:        awful.spawn.with_shell("scrot " .. ss_dirs .. "/Screenshot_%Y-%m-%d-%S.png" ..  "&& dunstify -i \"/home/saimoom/.config/awesome/images/screenshot/ss_full.png\" \"Screenshot\" \"saved!\"")
73:        awful.spawn.with_shell("scrot  -d 3" .. ss_dirs .. "/Screenshot_%Y-%m-%d-%S.png" ..  "&& dunstify -i \"/home/saimoom/.config/awesome/images/screenshot/ss_timed.png\" \"Screenshot\" \"saved!\"")

awesome/misc/rofi/theme.rasi
22:// import colors (change "saimoom" to your username)
23:@import "/home/saimoom/.config/awesome/misc/rofi/colors.rasi"

awesome/misc/scripts/Rofi/themes/four-horizontal.rasi
1:@import "/home/saimoom/.config/awesome/misc/scripts/Rofi/themes/colors.rasi"
```

**Comment by saimoomedits** [comment]
hey, thanks for letting me know!

should now be fixed in commit [5127593](https://github.com/saimoomedits/dotfiles/commit/51275938f9062691a1f5afb1df680a6c43dc7891)

**Comment by saimoomedits** [comment]
closing this issue
